scholia/app/static/scholia.js

```diff
@@ -539,3 +539,419 @@ function sparqlToShortInchiKey(sparql, key,  element, filename) {
     sparqlToDataTable2(new_sparql, element, filename);
 
 }
+};
+
+
+function workWithQ(q) {
+    var url = 'https://www.wikidata.org/w/api.php?action=wbgetentities&ids=' +
+	        q  + 
+	       '&format=json&callback=?';
+     
+    $.getJSON(url, function (data) {
+	 var item = data.entities[q];
+	 if ('en' in item.labels) {
+	     $("#h1").text(item.labels.en.value);
+	 }
+	 $("#h1").append(' (<a href="https://www.wikidata.org/wiki/' +  q  + ' ">' + q  + '</a>)');
+	 
+	 var detailsList = Array();
+	 try {
+	     var orcid = item.claims.P496[0].mainsnak.datavalue.value;
+	     detailsList.push( '<a href="https://orcid.org/"><img alt="' +
+			       'ORCID logo" src="{{ url_for(\'static\', filename=\'images/orcid_16x16.gif\') }}"' +
```

Comment:
This might set this PR back awhile, but you can't use any `{{ }}` code in the extracted JS. This is because the html is rendered as a jinja template and these values are replaced before being executed by the browser.

I would suggest either dropping the template (as you can do here - replace it with "/static/images/orcid_16x16.gif") or in the case of dynamic stuff like `{{ q }}`, make it a parameter to the function.

Reply:
PS, the above results in a `GET http://127.0.0.1:8100/author/%7B%7B%20url_for('static',%20filename='images/orcid_16x16.gif')%20%7D%7D 404 (NOT FOUND)` error in the console

Reply:
> This might set this PR back awhile, but you can't use any `{{ }}` code in the extracted JS. This is because only the HTML is rendered as a Jinja template, replacing these values before being executed in the browser.
> 
> I would suggest either dropping the template (as you can do here - replace it with "/static/images/orcid_16x16.gif") or in the case of dynamic stuff like `{{ q }}`, make it a parameter to the function.

Yeah, you right. I think you first recommendation is better because the GIF file is hard-coded

Reply:
@carlinmack I used a hybrid between the two solutions in case you want to change the static path and also in the case you want to add more files from the static path

![image](https://user-images.githubusercontent.com/1589267/128763588-691d6643-06df-4993-9cac-f8ffa7512b96.png)

Please, let me know what do you think about that.